src/core/fetch.cpp

```diff
@@ -226,6 +227,9 @@ namespace mamba
                 m_handle, CURLOPT_ACCEPT_ENCODING, "gzip, deflate, compress, identity");
             m_headers = curl_slist_append(m_headers, "Content-Type: application/json");
         }
+        m_headers = curl_slist_append(m_headers,
+                                      "User-Agent: Mamba/" MAMBA_VERSION_STRING
```

Comment:
Would you mind using `mamba/` (lowercase?)

Also Not sure how commmon it is to add a link to the website to the user agent. I think we can remove that. It would be cool though to add the cURL version using

`curl_version_info(CURLVERSION_NOW).version` (that returns a `char*`).